src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -80,13 +92,13 @@ const GuideStore = Reflux.createStore({
 
     // Select the first guide that hasn't been seen in this session and has all
     // required anchors on the page.
+    // If url hash is #assistant, show the first guide that has some required anchors.
```

Comment:
Comment is inaccurate, we still require all anchors

Reply:
fixed